spec/amadeus/namespaces.test.js

```diff
@@ -74,8 +74,8 @@ describe('Namespaces', () => {
       expect(amadeus.referenceData.location('ALHR').get).toBeDefined();
       expect(amadeus.referenceData.locations.get).toBeDefined();
       expect(amadeus.referenceData.locations.airports.get).toBeDefined();
-      expect(amadeus.referenceData.locations.pointsOfInterest.get).toBeDefined();
-      expect(amadeus.referenceData.locations.pointsOfInterest.bySquare.get).toBeDefined();
+      expect(amadeus.referenceData.locations.pointsOfInterest('XXX').get).toBeDefined();
```

Comment:
Can you replace XXX by an example?

Reply:
It doesn't matter as it's a test file.
Also, the whole file uses "XXX" in similar scenarios. Exception: locations API which was implemented long back.

Reply:
> It doesn't matter as it's a test file.

Yes and no, it is not because it works that it is the best, and it is not because it was done before that it is the reason to continue doing it. I agree with the fact that it works and it is fine, but I believe when possible, putting a real example for tests is always a good practice (and can help contributors). 

Not an important point anyways.